src/commands/cmd_profile.c

```diff
@@ -61,7 +61,7 @@ void Graph_Profile(void *args) {
 	}
 
 	result_set = NewResultSet(ctx, false);
```

Comment:
If not making the change suggested [here](https://github.com/RedisGraph/RedisGraph/pull/929/files?utf8=%E2%9C%93&diff=split&w=1#r382827623), `QueryCtx_SetResultSet` needs to be called.